lib/diagnostic.js

```diff
@@ -6,10 +6,17 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (string) {
+  let arrayOfWords = string.toUpperCase().split(' ');
+  console.log(arrayOfWords);
 };
 
+let string = "This is my Sentence!";
+
+normalizeWords(string);
+
+// use somehow /\w/ to get rid of punctation
+
```

Comment:
This is really well done, Esther. You totally get it, and you did a great job of checking your work.